ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -118,12 +118,15 @@ type wireframeBase = {
     /**
      * @description x and y are the top left corner of the element, if they are present then the element is absolutely positioned, if they are not present this is equivalent to setting them to 0
      */
-    x: number
-    y: number
+    x?: number
+    y?: number
     /*
-     * @description width and height are the dimensions of the element, the only accepted units is pixels. You can omit the unit.
+     * @description the width dimension of the element, either '100vw' i.e. viewport width. Or a value in pixels. You can omit the unit when specifying pixels.
+     */
+    width: number | '100vw'
```

Comment:
since I was allowing this for the custom event I thought I'd allow it here too.

We may never send it but 🤷